src/math/hyp1f1.hh

```diff
@@ -249,7 +251,9 @@ namespace rascal {
       }
 
      public:
-      size_t n_terms{0};
+      // Parameter used to fix the sum when evaluating the numerical
```

Comment:
Is that also documented somewhere else, I have a hard time following the reasoning here.

Reply:
don't understand this either, felix did this @felixmusil

Reply:
this parameter is used in the tests to fix the number of terms in the series expansion of the hyp1f1. It is useful when comparing the derivative with the numerical one so that the agreement is more consistent but it is not important anywhere else.